packages/twenty-front/src/modules/ui/input/components/SelectControl.tsx

```diff
@@ -15,21 +15,31 @@ const StyledControlContainer = styled.div<{ disabled?: boolean }>`
   display: flex;
   gap: ${({ theme }) => theme.spacing(1)};
   height: ${({ theme }) => theme.spacing(8)};
-  justify-content: space-between;
+  max-width: 100%;
   padding: 0 ${({ theme }) => theme.spacing(2)};
-`;
-
-const StyledControlLabel = styled.div`
-  align-items: center;
-  display: flex;
-  gap: ${({ theme }) => theme.spacing(1)};
+  flex-grow: 1;
+  flex-shrink: 1;
 `;
 
 const StyledIconChevronDown = styled(IconChevronDown)<{
   disabled?: boolean;
 }>`
   color: ${({ disabled, theme }) =>
     disabled ? theme.font.color.extraLight : theme.font.color.tertiary};
+  flex-shrink: 0;
+`;
+
+const StyledIconContainer = styled.span`
+  display: flex;
+  flex-shrink: 0;
+`;
+
+const StyledLabel = styled.span`
+  flex-grow: 1;
```

Comment:
is this flex-grow necessary?
This code looks very similar to OverflowingTextWithTooltip